src/point_cloud2_assembler.cpp

```diff
@@ -65,12 +76,10 @@ class PointCloud2Assembler : public BaseAssembler<sensor_msgs::PointCloud2>
     return (scan.width * scan.height);
   }
 
-  void ConvertToCloud(const string& fixed_frame_id, const sensor_msgs::PointCloud2& scan_in, sensor_msgs::PointCloud& cloud_out)
+  void Convert(const string& fixed_frame_id, const
+  sensor_msgs::PointCloud2& cloud_in, sensor_msgs::PointCloud2& cloud_out)
   {
-    sensor_msgs::PointCloud cloud_in;
-    sensor_msgs::convertPointCloud2ToPointCloud(scan_in, cloud_in);
-    tf_->transformPointCloud(fixed_frame_id, cloud_in, cloud_out) ;
-    return ;
+    pcl_ros::transformPointCloud(fixed_frame_id, cloud_in, cloud_out, *tf_);
```

Comment:
pcl is a huge dependency for that simple function. Use tf2_sensor_msgs instead: https://github.com/ros/geometry_experimental/blob/indigo-devel/tf2_sensor_msgs/include/tf2_sensor_msgs/tf2_sensor_msgs.h#L59